### ISSUE TITLE
ClamAV don't launch

### ISSUE THREAD

**Issue by arckosfr** [hardware/mailserver#28]
Amavis fail to start :
```bash
2016-07-07 08:16:16,105 INFO spawned: 'freshclam' with pid 259
2016-07-07 08:16:16,153 INFO exited: freshclam (exit status 40; not expected)
2016-07-07 08:16:17,158 INFO spawned: 'clamd' with pid 260
2016-07-07 08:16:17,195 INFO exited: clamd (exit status 1; not expected)
2016-07-07 08:16:19,622 INFO exited: freshclam (exit status 40; not expected)
2016-07-07 08:16:19,812 INFO gave up: freshclam entered FATAL state, too many st                              art retries too quickly
2016-07-07 08:16:20,881 INFO exited: clamd (exit status 1; not expected)
2016-07-07 08:16:21,883 INFO gave up: clamd entered FATAL state, too many start                               retries too quickly
```

That occur debug mail at each sending mail and no sending at all.

**Comment by arckosfr** [comment]
Maybe a solution : ?
```bash
root@ohmna2:~# docker exec -ti mailserver clamav
ERROR: Parse error at line 14: Unknown option AllowSupplementaryGroups
ERROR: Can't open/parse the config file /etc/clamav/freshclam.conf
freshclam: ERROR (abnormal termination)
clamd: ERROR (abnormal termination)
```

**Comment by arckosfr** [comment]
Resolved with remove AllowSupplementaryGroups in freshclam.conf and clamd.conf

**Comment by hardware** [comment]
ClamAV package was updated (AllowSupplementaryGroups option doesn't exist anymore), I will comment this two lines asap.

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=826406

**Comment by hardware** [comment]
Solved with `0.99.2+dfsg-0+deb8u2`
